<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>686</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1074</width>
    <height>686</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1074</width>
    <height>686</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestione Ricevute</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>686</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>GestisciRicevute.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="button_inserisci">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>100</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>INSERISCI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_elimina">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>190</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>ELIMINA</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>731</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="list_view">
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="GestisciRicevute.qrc"/>
 </resources>
 <connections/>
</ui>
